app/models/story.rb

```diff
@@ -41,14 +41,20 @@ class Story < ActiveRecord::Base
 
   def self.for_user_and_anime(user, anime, story_type="media_story")
     story = user.stories.where(story_type: story_type, target_id: anime.id, target_type: "Anime")
-    watchlist = Watchlist.find_by(user_id: user.id, anime_id: anime.id)
+    entry = LibraryEntry.find_by(user_id: user.id, anime_id: anime.id)
```

Comment:
To remove that if, what you think about what I did on https://github.com/DNA/hummingbird/blob/add-mangas-to-feed/app/models/story.rb#L57 ??